recipe/meta.yaml

```diff
@@ -1,36 +1,98 @@
-
+{% set sha256 = "cdb3860d1fbc5a5d628f985158da75fa82e17ceb24c2f0cd8cf5bc772c7e2e34" %}
+{% set paraview_sha = "98a2eea1af02a45fa6d9b72b97b67f991bcb1f0e" %}
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/OpenChemistry/{{ name }}/releases/download/{{ version  }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  - url: https://github.com/OpenChemistry/{{ name }}/releases/download/{{ version  }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+    folder: tomviz
+  - git_url: https://github.com/OpenChemistry/ParaView.git
+    git_rev: {{ paraview_sha }}
+    folder: paraview
 
 build:
   number: 0
 
 requirements:
   build:
     - cmake
+    - ninja
     - pkg-config
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
+    - {{ cdt('libx11-devel') }}          # [linux]
+    - {{ cdt('libxext-devel') }}         # [linux]
+    - {{ cdt('libxrender-devel') }}      # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libxau-devel') }}          # [linux]
+    - {{ cdt('alsa-lib-devel') }}        # [linux]
+    - {{ cdt('gtk2-devel') }}            # [linux]
+    - {{ cdt('gtkmm24-devel') }}         # [linux]
+    - {{ cdt('libdrm-devel') }}          # [linux]
+    - {{ cdt('libxcomposite-devel') }}   # [linux]
+    - {{ cdt('libxcursor-devel') }}      # [linux]
+    - {{ cdt('libxi-devel') }}           # [linux]
+    - {{ cdt('libxrandr-devel') }}       # [linux]
+    - {{ cdt('pciutils-devel') }}        # [linux]
+    - {{ cdt('libxscrnsaver-devel') }}   # [linux]
+    - {{ cdt('libxtst-devel') }}         # [linux]
+    - {{ cdt('libselinux-devel') }}      # [linux]
+    - {{ cdt('libxdamage') }}            # [linux]
+    - {{ cdt('libxdamage-devel') }}      # [linux]
+    - {{ cdt('libxfixes') }}             # [linux]
+    - {{ cdt('libxfixes-devel') }}       # [linux]
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - {{ cdt('libxrandr') }}             # [linux]
+    - {{ cdt('alsa-lib') }}              # [linux]
+    - {{ cdt('xorg-x11-server-xvfb') }}  # [linux]
+    - {{ cdt('libxt-devel') }}           # [linux]
+    - {{ cdt('libsm-devel') }}           # [linux]
+    - {{ cdt('libice-devel') }}          # [linux]
+    - {{ cdt('libuuid-devel') }}         # [linux]
   host:
-    - python
+    - python=3.7
+    - hdf5
+    - libpng
+    - jpeg
+    - libtiff
+    - libxml2
+    - zlib
+    - freetype
+    - libprotobuf
+    - lz4-c
+    - expat
+    - libnetcdf
+    - glew
+    - double-conversion
+    - jsoncpp
+    - xz
+    - cgns
+    - qt
+    - pugixml
+    - libtheora
+    - gl2ps
+    - tbb-devel
   run:
-    - python
+    - python=3.7
     - numpy
+    - scipy
+    - itk
+    - marshmallow
+    - jsonpointer
 
 test:
   imports:
-    - tomviz
+    - paraview
```

Comment:
Is it supposed to be `paraview` here?

Reply:
... the test is failing on importing it: https://dev.azure.com/nsls2forge/nsls2forge/_build/results?buildId=5742&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f&l=433.

Reply:
Yes, for it just paraview. There isn't really a tomviz package to test import as its a Qt app. We can refine the testing later. The test is failing on Windows as we currently aren't building anything on Window. This just has the bare bones for Linux.